ci/templates/appveyor.yml

```diff
@@ -22,6 +22,9 @@ install:
   - echo PyPy installed
   - pypy --version
 
+  # Upgrade virtualenv for e.g. more-itertools to be handled properly.
+  # Pin it to work around https://github.com/tox-dev/tox/issues/1389.
+  - C:\Python35\python -m pip install -U virtualenv==16.5.0
```

Comment:
fix going out under https://dev.azure.com/pypa/virtualenv/_build/results?buildId=11337

Reply:
https://pypi.org/project/virtualenv/16.7.3/

Reply:
Thanks, @gaborbernat.
@graingert 
I think we can remove this then.

Reply:
@blueyed @gaborbernat I've removed this from the template and the generated appveyor.yml

Reply:
ok that broke it - I think it would be best to go back to what I had before

Reply:
An upgrade in general might be needed.
(there's no link to a build log so I cannot really tell)